client/my-sites/site-settings/theme-enhancements.jsx

```diff
@@ -77,6 +78,40 @@ class ThemeEnhancements extends Component {
 	}
 
 	renderInfiniteScrollSettings() {
+		if ( this.props.jetpackSettingsUI ) {
+			return this.renderJetpackInfiniteScrollSettings();
+		}
+		return this.renderSimpleSiteInfiniteScrollSettings();
+	}
+
+	renderSimpleSiteInfiniteScrollSettings() {
+		const { translate } = this.props;
+		return (
+			<FormFieldset>
+				<FormLegend>{ translate( 'Infinite Scroll' ) }</FormLegend>
+
+				<div className="theme-enhancements__info-link-container site-settings__info-link-container">
+					<InfoPopover position="left">
+						<ExternalLink
+							href="https://support.wordpress.com/infinite-scroll/"
+							icon
+							target="_blank"
+						>
+							{ translate( 'Learn more about Infinite Scroll.' ) }
```

Comment:
Monir: In the desktop tooltip, the length of the sentence makes it so that the link icon is wrapping to the next line alone. Wondering if we'd want to change the copy so the icon is not alone on the second line. Something like 'Click to learn more about Infinite Scroll.'. No need to change though.